NewForce-Cohort-5/nutshell-nitro-menance-overboard
# Bg messages css

# Description
Added an edit form for messages. Edit form button should only be visible to each users own messages posted. Did some small changes to css and edited Article.js to open links in new tabs.
## Type of change
# Testing Instructions
Log in and see if you're able to edit any messages you have posted under that login info.
# Checklist:
